14.0.Dockerfile

```diff
@@ -35,8 +37,25 @@ ENV DB_FILTER=.* \
 # See https://github.com/$ODOO_SOURCE/blob/$ODOO_VERSION/debian/control
 RUN apt-get -qq update \
     && apt-get install -yqq --no-install-recommends \
-        curl \
-    && curl -SLo wkhtmltox.deb https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox_${WKHTMLTOPDF_VERSION}-1.buster_amd64.deb \
+        curl; \
+    if [ "$TARGETARCH" = "arm64" ]; then \
+        if [ "$WKHTMLTOPDF_VERSION" != "0.12.6.1" ]; then \
```

Comment:
How is this supposed to work? As in line 7 the version is hard coded.
`ARG WKHTMLTOPDF_VERSION=0.12.5`